### DISCUSSION TITLE
crop function medialibrary 11

### DISCUSSION THREAD

**Discussion by Kenny201** [spatie/laravel-medialibrary#3510] [Q&A] [answered]
Can you explain why the crop function crops images differently? I'm using the function in medialibrary 10 ->crop('300', '500', 'crop-center'). But when I use a similar version from medialibrary 11, it crops completely differently, not like in the previous version

**Comment by freekmurze** [comment]
Could you take a look @timvandijck

**Comment by timvandijck** [comment]
@Kenny201 would it be possible to provide an example of an original image, the result and what you would expect when it works properly?

**Reply by Kenny201** [reply]
Here is the original photo without cropping
https://ibb.co/KDg0MSq

**Reply by Kenny201** [reply]
here is a photo cropped by medialibrary 10
<img width="310" alt="image" src="https://github.com/spatie/laravel-medialibrary/assets/24923249/2724f105-fb2d-4cee-bc46-079ac76b6c18">

**Reply by Kenny201** [reply]
here is a photo cropped by medialibrary 11
<img width="309" alt="image" src="https://github.com/spatie/laravel-medialibrary/assets/24923249/87f5afdf-4203-4fe8-9631-b1bb4e65c627">

**Reply by Kenny201** [reply]
I expected it to be like in medialibrary 10

**Reply by Kenny201** [reply]
in medialibrary 10 I use the function
             ->crop('crop-center', 400, 516)

**Reply by Kenny201** [reply]
in medialibrary 11 I use the function
             ->crop(400, 516)

**Comment by Kenny201** [comment, marked as answer]
Got it, can use it
->fit(Fit::Crop,400, 516)
